server/transaction_commit_informer.go

```diff
@@ -25,14 +25,18 @@ import (
 	"context"
 
 	"github.com/cloudwan/gohan/schema"
+	"sync"
 )
 
-var transactionCommited chan int
+var (
+	transactionCommited chan int
+	wasCommited         sync.Once
```

Comment:
What was commited? this name is not self-explaining. Please rename this to something more meaningful, like transactionCommitedOnce or similar